src/plugins/visualizations/public/wizard/search_selection/search_selection.tsx

```diff
@@ -24,13 +24,15 @@ import React from 'react';
 import { IUiSettingsClient, SavedObjectsStart } from '../../../../../core/public';
 
 import { SavedObjectFinderUi } from '../../../../../plugins/saved_objects/public';
-import { VisType } from '../../vis_types';
+import type { VisType } from '../../vis_types';
```

Comment:
👍  in `Visualization Team `you are the first person who started to use import type ❤️

Reply:
Didn't know about this!